ws/src/main/scala/de/thm/ii/fbs/services/conferences/JitsiService.scala

```diff
@@ -18,26 +19,38 @@ import org.springframework.stereotype.Service
   */
 @Service
 class JitsiService(templateBuilder: RestTemplateBuilder) extends ConferenceService {
-  /**
-    * The name of the conference service
-    */
-  val name = "jitsi"
-
   /**
     * Creates a new Conference using Jitsi
-    * @param cid the id for the new conference
+    * @param courseId the id for the new conference
     * @return the newly created conference
     */
-  override def createConference(cid: String): Conference = {
+  override def createConference(courseId: String): Conference = {
+    val cid = UUID.randomUUID().toString
     val uri = this.registerJitsiConference(cid)
     new Conference {
       override val id: String = cid
-      override val serviceName: String = JitsiService.this.name
-      private val meetingURL = uri
+      override val serviceName: String = JitsiService.name
+      override val courseId: String = courseId
+      private val meetingURL: URI = uri
 
       override def getURL(user: User, moderator: Boolean): URI = meetingURL
       override def toMap: Map[String, String] = Map("id" -> id, "service" -> serviceName)
-    }
+
+      /**
+        * The visibility of the Conference
+        */
+      override val visibility: String = "false"
```

Comment:
* This must be a var because `visibility` in Conference is also a var.
* It should be next to the other propertie decleration and should not have a comment.